app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt

```diff
@@ -285,6 +285,12 @@ class ChatActivity : AppCompatActivity(), IChatView {
             }
         }
 
+
+    }
```

Comment:
What is being closed here?

Reply:
the code was written between the last brace and so it felt as though there was an extra brace, it's resolved in the latest commit